nestkernel/nest_time.h

```diff
@@ -509,14 +522,26 @@ class Time
   delay
   get_steps() const
   {
-    if ( tics == LIM_POS_INF.tics )
-      return LIM_POS_INF.steps;
-    if ( tics == LIM_NEG_INF.tics )
-      return LIM_NEG_INF.steps;
+    return steps_;
+  }
 
-    // round tics up to nearest step
-    // by adding TICS_PER_STEP-1 before division
-    return ( tics + Range::TICS_PER_STEP_RND ) / Range::TICS_PER_STEP;
+  void
+  calculate_steps()
+  {
+    if ( tics == LIM_POS_INF.tics )
```

Comment:
Is that safer? I think an assert for `tics <= ...` should be there -- it should never be greater than the limit, that's just as broken as missing it altogether.